docs/docs/unit-testing.md

```diff
@@ -318,3 +283,8 @@ though remember you may need to install the Babel 7 versions. See
 
 For more information on Jest testing, visit
 [the Jest site](https://jestjs.io/docs/en/getting-started).
+
+For an example encapsulating all of these techniques--and a full unit test suite with [react-testing-library][react-testing-library], check out the [using-jest][using-jest] example.
+
+[using-jest]: examples/using-jest
```

Comment:
```suggestion
[using-jest]: https://github.com/gatsbyjs/gatsby/tree/master/examples/using-jest
```
Need to be absolute as we will be linking from gatsbyjs.org

Reply:
Thanks @pieh!